### PR TITLE
doc - Change WeaklyUp is used in DD #21502

### PR DESCRIPTION
**Pull request by jgordijn**

### CODE REVIEW

File: akka-docs/rst/scala/distributed-data.rst [hunk 1]
```diff
@@ -41,8 +41,10 @@ with a specific role. It communicates with other ``Replicator`` instances with t
 ``akka.cluster.ddata.DistributedData`` extension.
 
 Cluster members with status :ref:`WeaklyUp <weakly_up_scala>`, if that feature is enabled,
-will currently not participate in Distributed Data, but that is something that should be possible to
-add in a future release.
+will participate in Distributed Data. Note that it will not participate in any actions where the
```
Comment (patriknw) [inline akka-docs/rst/scala/distributed-data.rst]: between those two sentences something like: "The data will be replicated to the `WeaklyUp` nodes with the background gossip protocol."

**Comment by patriknw** [comment]
thanks for fixing, @jgordijn

**Comment by johanandren** [comment]
There's a `akka-docs/rst/java/distributed-data.rst` that should have the same update.

**Comment by johanandren** [comment]
Refs #21502

**Comment by jgordijn** [comment]
Processed both review comments of @patriknw and @johanandren

**Review by johanandren — APPROVED** [review]
LGTM, thanks Jeroen!

**Comment by patriknw** [comment]
Thanks, @jgordijn

**Pull request merged**